app/src/main/java/org/wikipedia/settings/Prefs.kt

```diff
@@ -712,4 +713,23 @@ object Prefs {
     var placesWikiCode
         get() = PrefsIoUtil.getString(R.string.preference_key_places_wiki_code, WikipediaApp.instance.appOrSystemLanguageCode).orEmpty()
         set(value) = PrefsIoUtil.setString(R.string.preference_key_places_wiki_code, value)
+
+    var placesLastLocation: Location?
```

Comment:
Should we also save the last Zoom level?

Reply:
Yes and done!